vizro-core/examples/kpi/README.md

```diff
@@ -0,0 +1,15 @@
+# KPI dashboard
+
+This demo dashboard provides an example of a Key Performance Indicator (KPI) dashboard, designed to help users get started and extend further.
+It uses fictional budget data to demonstrate the capabilities of Vizro using real world applications.
+
+Special thanks to the [#RWFD Real World Fake Data initiative](https://opendatainitiative.io/), a community project that
+provides high-quality fake data for creating realistic dashboard examples for real-world applications.
+
+Note: The data has been additionally edited for the purpose of this example.
+
+## How to run the example locally
+
+1. Download the `Financial Consumer Complaints.csv` file from [this link](https://data.world/markbradbourne/rwfd-real-world-fake-data/workspace/file?filename=Financial+Consumer+Complaints.csv) and place it in the `data` folder
+2. If you have `hatch` set up, run the example with the command `hatch run example kpi`. Otherwise, run the `app.py` file with your environment activated where `vizro` is installed.
+3. You should now be able to access the app locally via http://127.0.0.1:8050/.
```

Comment:
Definitely we need an easier way for people to get the code/run the examples in general if we actually think someone would want to do so, because for someone to run it at the moment would need them to download the whole `kpi` folder which I think is actually not that easy from Github UI?

But that's way beyond the scope of this PR - just thinking out loud.

Reply:
Yes, I wanted to discuss that with @maxschulz-COL and you on Wednesday. I have many new examples coming; optimally, I don't want them to all live in a separate folder. Some can live in our public repository; and for some, it makes more sense to add them to my private demo, as I want to get quick and dirty examples out as well.

But the hosting topic and where this should all live is a big question for me

Reply:
I have started exploring options, let's discuss on Wednesday where we stand. Have a few ideas on what we could do.